packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -33,14 +33,20 @@ set timeDilation(double value) {
   _timeDilation = value;
 }
 
-/// A frame-related callback from the scheduler.
+/// Signature for frame-related callbacks from the scheduler.
 ///
 /// The timeStamp is the number of milliseconds since the beginning of the
 /// scheduler's epoch. Use timeStamp to determine how far to advance animation
```

Comment:
nit: while you're here could you wrap "timeStamp" in backticks?

Reply:
good catch. done.